lib/timers.js

```diff
@@ -118,6 +124,11 @@ function enroll(item, msecs) {
  * DOM-style timers
  */
 
+function lazyDOMException(message) {
+  if (DOMException === undefined)
+    DOMException = require('internal/per_context/domexception').DOMException;
```

Comment:
The `per_context/` modules are not expected to be required (currently) afaik – you can get `DOMException` from `internalBinding('messaging')` instead?
```suggestion
    DOMException = internalBinding('messaging').DOMException;
```